lua/lspconfig/solargraph.lua

```diff
@@ -30,9 +30,7 @@ You can install solargraph via gem install.
 gem install --user-install solargraph
 ```
     ]],
-    default_config = {
-      root_dir = [[root_pattern("Gemfile", ".git")]],
```

Comment:
`Gemfile` as a fallback here made more sense that not having it.

Reply:
This is documentation only, it's autogenerated now and does not affect current behavior in any way.